### ISSUE TITLE
Neuron

### ISSUE THREAD

**Issue by bitekas** [Templarian/MaterialDesign#6234]
### Before submitting a new icon request, please confirm the following:

- [X] I have [searched all issues](https://github.com/Templarian/MaterialDesign/issues) to make sure there isn't a request for this icon.
- [X] I have [searched the current library](https://materialdesignicons.com/) to make sure the icon doesn't already exist.
- [X] I understand that [brand icons are no longer being added](http://dev.materialdesignicons.com/roadmap/brand-icons) and I can use [SimpleIcons](https://simpleicons.org/) for my brand icon needs.
- [X] I only intend to request a single icon (or a few near-identical ones) in this issue.

### What would you use this icon for?

Neuron to represent neural network

### Give us an example of what this icon might look like.

A neuron cell with visible axons and nucleus

**Comment by KerkDay** [comment]
![image](https://user-images.githubusercontent.com/26054280/140257273-f9395f94-70c9-49b9-ba3e-931788ca9a95.png)
[neuron.zip](https://github.com/Templarian/MaterialDesign/files/7472292/neuron.zip)

**Comment by Circeus** [comment]
I don't think a literal neuron is the better choice for "neural network", which is what the request is really about. Looking at noun project, I see more promise in the "connected half brain" concept:
https://thenounproject.com/term/neural-network/1870007/
https://thenounproject.com/term/neural-network/3732456/
(this "neural chip" interpretation is also pretty nice: https://thenounproject.com/term/neural-network/2271602/)

**Comment by bitekas** [comment]
@Circeus True, the request is about the neural network rather than a neuron cell. I like the work in progress version which is clearly about neurons. Maybe incoming and outgoing connections could be added to make it look more like a network?

https://thenounproject.com/term/neural-network/1870007/
https://thenounproject.com/term/neural-network/3732456/  - these two are great illustrations for for brain interface (Neuralink)
https://thenounproject.com/term/neural-network/2271602/ - this is great for microchips

**Comment by Simran-B** [comment]
Google has an icon called `network-intelligence`:

![image](https://user-images.githubusercontent.com/7819991/140590587-930b0420-f693-4809-adfc-b54f7692b609.png)

**Comment by goyney** [comment]
We also already have `brain`. 
[![brain](https://materialdesignicons.com/icon/brain)](https://materialdesignicons.com/icon/brain)

**Comment by Simran-B** [comment]
The existing icon depicts an actual brain whereas the Google icon tries to depict an artificial brain for a machine learning context
